src/platform/packages/private/kbn-reporting/public/index.ts

```diff
@@ -26,10 +28,13 @@ import type { SharePluginStart } from '@kbn/share-plugin/public';
 export interface KibanaContext {
   http: CoreSetup['http'];
   application: CoreStart['application'];
+  settings: CoreStart['settings'];
   uiSettings: CoreStart['uiSettings'];
   docLinks: CoreStart['docLinks'];
   data: DataPublicPluginStart;
   share: SharePluginStart;
+  actions: ActionsPublicPluginSetup;
```

Comment:
We need the actions plugin to get the `validateEmailAddresses` utility (not ideal, but in the future we could export that to a more generic place)